src/erasure/ErasurePost.js

```diff
@@ -55,11 +56,71 @@ class ErasurePost {
    * @method proofhash
    * @returns {string} bytes32 sha256 proofhash
    * @returns {string} base58 multihash format of the proofhash
+   * @returns {Promise<Object>} proof containing content of post
    */
   proofhash = () => {
     return {
       proofhash: this.#proofhash,
-      multihash: Utils.sha256ToHash(this.#proofhash)
+      multihash: Utils.sha256ToHash(this.#proofhash),
+      getProof: () => {
+        return this.parseProof(this.#proofhash);
+      }
+    };
+  };
+
+  parseProof = async proofhash => {
+    try {
+      var proofBlob = await IPFS.get(Utils.sha256ToHash(proofhash));
```

Comment:
Can you change the `var`?
Always use either `let` or `const` based on usage (in this case, you need the `let` above the `try`)